.vale.ini

```diff
@@ -0,0 +1,19 @@
+# where styles are saved
+StylesPath = .vale/styles/
+
+# the minimum alert level
+MinAlertLevel = suggestion
+
+# style guide packages
+Packages = Google, write-good
+
+# map MDX files to Markdown since Vale doesn't natively support MDX
+[formats]
+mdx = md
+
+Vocab = Base
+
+
+# apply Google and Novu style guide rules to both Markdown (.md) and MDX (.mdx) files
+[*.{md,mdx}]
+BasedOnStyles = Google, Novu, write-good
```

Comment:
@Aviatorscode2 

Could you please share the link of Novu? I could not find here
https://vale.sh/explorer